core/src/main/scala/kafka/server/ZkAdminManager.scala

```diff
@@ -158,7 +158,7 @@ class ZkAdminManager(val config: KafkaConfig,
 
         val nullConfigs = topic.configs.asScala.filter(_.value == null).map(_.name)
         if (nullConfigs.nonEmpty)
-          throw new InvalidRequestException(s"Null value not supported for topic configs : ${nullConfigs.mkString(",")}")
+          throw new ConfigException(s"Null value not supported for topic configs: ${nullConfigs.mkString(",")}")
```

Comment:
Hmm, which error code does this get converted to in the `CreateTopics` response?

Reply:
I was going to suggest using InvalidConfigurationException instead, but we might need to be a little careful. I am not sure if INVALID_CONFIG is already an expected error for `CreateTopics`.

Reply:
I think `InvalidConfigurationException`. https://github.com/apache/kafka/blob/f36de0744b915335de6b636e6bd6b5f1276f34f6/core/src/main/scala/kafka/server/ZkAdminManager.scala#L217

Reply:
Yeah, `INVALID_CONFIG ` is not used in `CreateTopics` currently, I changed it to `InvalidConfigurationException`.